src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -371,10 +401,10 @@ def full_dict(
         event_id = self.event["id"]
         # We're in a loop!
```

Comment:
🤔  Why is this comment here?

Reply:
This refers to a loop in the trace structure lol

Reply:
Do we really need this here?

Reply:
Hmm probably could reword it to be better